api/src/http.rs

```diff
@@ -398,32 +590,34 @@ lazy_static! {
             routes: HashMap::new(),
         };
 
-        // Global
+        // Common
         r.routes.insert(endpoint_v1!("/daemon"), Box::new(InfoHandler{}));
-        r.routes.insert(endpoint_v2!("/daemon"), Box::new(InfoHandler{}));
-        r.routes.insert(endpoint_v1!("/daemon/backend"), Box::new(FsBackendInfo{}));
-        r.routes.insert(endpoint_v2!("/daemon/backend"), Box::new(FsBackendInfo{}));
+        r.routes.insert(endpoint_v2!("/daemon"), Box::new(InfoV2Handler{}));
         r.routes.insert(endpoint_v1!("/daemon/events"), Box::new(EventsHandler{}));
         r.routes.insert(endpoint_v2!("/daemon/events"), Box::new(EventsHandler{}));
```

Comment:
Can we remove these duplicated APIs from v2? If anyone wants to use them, they should just use the v1 APIs.